docs/src/examples/RateSystem.md

```diff
@@ -0,0 +1,101 @@
+# Studying R-Tipping
+
+Let us explore a simple prototypical example of how to use the R-tipping functionality of this package.
+We start with defining an autonomous deterministic dynamical system (i.e. a `CoupledODEs`). Then, we define a time-dependent forcing protocol called `RateConfig`, describing how the parameters of the previously defined autonomous `CoupledODEs` will be changed over time. For times `t` smaller than the time `t_start`, the system is autonomous, then non-autonomous for `t_start < t < t_start + ramp_t_length` with the paramter ramping given by the `RateConfig` and for `t > t_start + ramp_t_length` the system is autonomous again. This setting is a widely used and convenient for studying R-tipping.
+
+We first consider the following simple one-dimensional autonomous system with one attractor, given by the ordinary differential equation:
+```math
+\begin{aligned}
+    \dot{x} &= (x+p)^2 - 1
+\end{aligned}
+```
+The parameter ``p`` shifts the location of the extrema of the drift field. 
+We implement this system as follows:
+
+```@example RateSystem
+using CriticalTransitions
+using CairoMakie
+
+function f(u,p,t) # out-of-place
+    x = u[1]
+    dx = (x+p[1])^2 - 1
+    return SVector{1}(dx)
+end
+
+x0 = [-1.]
+auto_sys = CoupledODEs(f,x0,[0.0]);
+```
+
+## Applying the parameter ramping
+
+Now, we want to explore a non-autonomous version of the system by applying a parameter ramping. 
+As discussed, we consider a setting where in the past and in the future the system is autnonomous and in between there is a non-autonomous period `[t_start, t_start+ramp_t_length]` with a time-dependent parameter ramping given by the function ``p(t)``. Choosing different values of the parameter `ramp_t_length` allows us to vary the speed of the parameter ramping.
+
+We start by defining the function `p(p_parameters, t)`:
+```@example RateSystem
+function p(p_parameters, t)
+    p_max = p_parameters[1]
+    p_ = (p_max/2)*(tanh(p_max*t/2)+1)
+    return SVector{1}(p_)
+end
```

Comment:
I don't agree with this design. I think the function `p` should not have parameters. It should just be `p(t)`. Also, we should name it something else, as it missleadingly represents the universal symbol for parameters, even though it isn't a parameter at all. It is just a profile. We should name it `profile(t)`.

Reply:
What's the reason for the parameters? It defnitely isn't to control the rate/steepness of the profile, because this is controlled externally with the dp, dt arguments.